packages/vuetify/src/composables/density.ts

```diff
@@ -5,7 +5,7 @@ import { getCurrentInstanceName, propsFactory } from '@/util'
 // Types
 import type { PropType } from 'vue'
 
-const allowedDensities = [null, 'default', 'comfortable', 'compact'] as const
+export const allowedDensities = [null, 'default', 'comfortable', 'compact'] as const
```

Comment:
Doing this because want to centralise densities array and reused in each select components spec file. Is it necessary and right approach?

Same question for variants